src/@commonsku/styles/Popup.tsx

```diff
@@ -114,7 +114,7 @@ const PopupContainer: React.FC<React.PropsWithChildren<{}>> = ({ children }) =>
 }
 
 export type PopupProps = React.PropsWithChildren<{
-  header?: React.Component,
+  header?: React.Component | React.FC | Element | React.ReactNode,
```

Comment:
I think this should probably just be `React.ReactNode`.  Does that work? Can you verify?

Reply:
Yeah it is good to just use `React.ReactNode`